lib/auth.ts

```diff
@@ -133,6 +147,81 @@ export class Auth {
     return token;
   };
 
+  /**
+   * Perform a request against Kontist REST API
+   */
+  private request = async (path: string, method: HttpMethod, body?: string) => {
+    if (!this.token) {
+      throw new Error("User unauthorized");
+    }
+
+    const requestUrl = new URL(path, this.baseUrl).href;
+
+    const response = await fetch(requestUrl, {
```

Comment:
should we throw an error when `response.ok === false`?

Reply:
Great idea, I have added this check here: ed834c70d4864e03a6d94fe79925dd8193815249